pkg/platform/aws/aws_internal_test.go

```diff
@@ -222,3 +223,64 @@ func TestConfigurationIsValidWhen(t *testing.T) {
 		})
 	}
 }
+
+func Test_config_checkCPUManagerPolicy(t *testing.T) {
+	tests := []struct {
+		name        string
+		WorkerPools []workerPool
+		want        hcl.Diagnostics
+	}{
+		{
+			name: "valid_cpu_manager_policy_values",
+			WorkerPools: []workerPool{
+				{CPUManagerPolicy: ""},
+				{CPUManagerPolicy: "static"},
+				{CPUManagerPolicy: "none"},
+			},
+			want: nil,
+		},
+		{
+			name: "invalid_cpu_manager_policy_value",
+			WorkerPools: []workerPool{
+				{CPUManagerPolicy: ""},
+				{Name: "foobar", CPUManagerPolicy: "foobar"},
+			},
+			want: hcl.Diagnostics{&hcl.Diagnostic{
+				Severity: hcl.DiagError,
+				Summary:  "invalid cpu_manager_policy",
+				Detail:   "Worker pool 'foobar' has invalid cpu_manager_policy 'foobar'",
+			}},
+		},
+		{
+			name: "all_invalid_values",
+			WorkerPools: []workerPool{
+				{Name: "foo", CPUManagerPolicy: "foo"},
+				{Name: "bar", CPUManagerPolicy: "bar"},
+			},
+			want: hcl.Diagnostics{
+				&hcl.Diagnostic{
+					Severity: hcl.DiagError,
+					Summary:  "invalid cpu_manager_policy",
+					Detail:   "Worker pool 'foo' has invalid cpu_manager_policy 'foo'",
+				},
+				&hcl.Diagnostic{
+					Severity: hcl.DiagError,
+					Summary:  "invalid cpu_manager_policy",
+					Detail:   "Worker pool 'bar' has invalid cpu_manager_policy 'bar'",
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			c := &config{
```

Comment:
```suggestion
		t.Run(tt.name, func(t *testing.T) {
		        t.Parallel()
		        
			c := &config{
```

Reply:
Can't make it parallel twice:

```bash
go test -timeout 30s -run TestCheckCPUManagerPolicy github.com/kinvolk/lokomotive/pkg/platform

--- FAIL: TestCheckCPUManagerPolicy (0.00s)
panic: testing: t.Parallel called multiple times [recovered]
	panic: testing: t.Parallel called multiple times

goroutine 20 [running]:
testing.tRunner.func1.2(0x7f3f00, 0xa441b0)
	/usr/local/go/src/testing/testing.go:1144 +0x332
testing.tRunner.func1(0xc00008a900)
	/usr/local/go/src/testing/testing.go:1147 +0x4b6
panic(0x7f3f00, 0xa441b0)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
testing.(*T).Parallel(0xc00008a900)
	/usr/local/go/src/testing/testing.go:1036 +0x36f
github.com/kinvolk/lokomotive/pkg/platform_test.TestCheckCPUManagerPolicy(0xc00008a900)
	/home/hummer/work/lokomotive/pkg/platform/platform_test.go:236 +0x569
testing.tRunner(0xc00008a900, 0x9d2a90)
	/usr/local/go/src/testing/testing.go:1194 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1239 +0x2b3
FAIL	github.com/kinvolk/lokomotive/pkg/platform	0.016s
FAIL
```

Reply:
My bad I had put it in the for loop out of `t.Run` :laughing: